examples/example_otap.py

```diff
@@ -35,72 +56,131 @@
 
     parser.add_argument('--network',
                         type=int,
+                        required=True,
                         help="Network address concerned by scratchpad")
     parser.add_argument('--file',
                         help="Scratcphad to use")
-    parser.add_argument("--gateway",
+
+    mutual_exclusive_group_1 = parser.add_mutually_exclusive_group()
+    mutual_exclusive_group_1.add_argument("--gateway",
                         type=str,
```

Comment:
If gateway is supposed to be a list (according to the help), someone might try to use something like `--gateway gw123,gw4253` in the command. But, there is nothing that parse this string. It means that it will recognize one gateway named "gw123,gw4253". Therefore, string type should be changed.

Reply:
You are correct.
Space separator is expected (as defined by 'nargs'). Added explanation to help